src/components/Icons/index.tsx

```diff
@@ -55,12 +59,26 @@ export default function Icon({
       }
     });
   }
+  if (url && tag === 'a') {
+    Object.assign(iconProps, {
+      href: url
+    });
+    if (replace) {
+      Object.assign(iconProps, {
+        target: '_self'
+      });
+    } else {
+      Object.assign(iconProps, {
+        target: '_blank'
```

Comment:
these don't belong here

Reply:
ok. I just think maybe one day we need the icon to be an 'a' tag. I will delete these codes.